docs/src/main/asciidoc/doc-reference.adoc

```diff
@@ -290,6 +307,45 @@ examples:
 // Source: integration-tests/micrometer-prometheus/src/main/java/io/quarkus/doc/micrometer/ExampleResource.java
 ----
 
+== Document attributes and variables
+
+=== Categories
+
+Quarkus documentation is grouped into the following categories. 
+
+.Categories
+[cols="<m,<2",options="header"]
+|===
+|Category | Description
+| alt-languages | Support for other languages, namely Kotlin and Scala
+| architecture | Quarkus runtime and ecosystem architecture
+| business-automation | Business automation integrations
+| cloud | Integrations and support for cloud services
+| command-line | Command Line Applications
+| compatibility | Compatibility with other languages and frameworks 
+| contributing | Guidance and references to help you contribute to Quarkus.
+| core | Information about how the Quarkus works
+| data | Topics related to using data sources with Quarkus
+| getting-started | Getting started materials
+| integration | Support for integration extensions (Camel)
+| messaging | Integrations with messaging systems like Kafka, AMQP, or RabbitMQ.
+| miscellaneous | Miscellaneous
+| observability | Extensions and integrations for runtime and application observability
+| reactive | Extensions that support reactive technologies and techniques
+| security | Security
+| serialization | Serialization
+| tooling | Tooling
+| web | Web
+| writing-extensions | Writing Extensions
+|===
+
+Document categories are specified in the <<Document header>> as a comma-separated value: 
```

Comment:
Suggestion to convert this into a more active and action-oriented step:

. Tag your content to improve findability by adding at least one category to the `categories` attribute line in the document header. To add multiple categories, use comma-separated values. For example: